eventually/vote/models.py

```diff
@@ -0,0 +1,230 @@
+"""
+Vote & Answer models
+====================
+
+This module implements class that represents the vote and answer models
+"""
+from django.db import models, IntegrityError
+
+
+class Vote(models.Model):
+    """
+        ..class::Vote
+
+        Create Vote model
+
+        Attributes:
+        ===========
+
+            :param is_active: active vote or end
+            :type is_active: BooleanField
+
+            :param is_extended:
+            :type is_extended: BooleanField
+
+            :param title: title of vote
+            :type title: CharField
+
+            :param vote_type: type of vote - we can choose one answer or many
+            :type vote_type: CharField
+
+            :param create_at:
+            :type create_at: TimeField
+
+            :param update_at:
+            :type update_at: TimeField
+    """
+
+    MULTI_CHOICES = (
+        ('O', 'One'),
+        ('M', 'Multi'),
+    )
+
+    is_active = models.BooleanField(default=True)
+    is_extended = models.BooleanField(default=True)
+    title = models.TextField(max_length=100)
+    vote_type = models.CharField(max_length=2, choices=MULTI_CHOICES)
+    create_at = models.TimeField(auto_now_add=True, editable=False)
+    update_at = models.TimeField(auto_now=True)
+
+    def to_dict(self):
+        """
+        Method that return Vote object in dictionary
+
+        :return:models-fields in dictionary
+        """
+        return {
+            'is_active': self.is_active,
+            'is_extended': self.is_extended,
+            'title': self.title,
+            'type': self.vote_type,
+            'create_at': self.create_at,
+            'update_at': self.update_at
+        }
+
+    @staticmethod
+    def get_by_id(vote_id):
+        """
+        Static method that return Vote object by id
+
+        :param id(IntegerField): id of element in model
+        :return: object with element, searched by id
+        """
+        try:
+            return Vote.objects.get(id=vote_id)
+        except Vote.DoesNotExist:
+            return None
+
+    @staticmethod
+    def create(my_is_active, my_is_extended, my_title, my_vote_type):
+        """
+        Static method that create new Vote object
+
+        :param my_is_active(BooleanField): active vote or end
+        :param my_is_extended(BooleanField):
+        :param my_title(CharField): title of vote
+        :param my_vote_type(CharField): type of vote - we can choose one answer or many
+        :return: new created object or None
+        """
+        vote = Vote(is_active=my_is_active,
+                    is_extended=my_is_extended,
+                    title=my_title,
+                    vote_type=my_vote_type)
+
+        try:
+            vote.save()
+            return vote
+        except ValueError:
+            return None
+
+    def update(self, my_is_active, my_is_extended, my_title, my_vote_type):
```

Comment:
the prefix 'my_', add the default value if needed

Reply:
done